README.md

```diff
@@ -2,3 +2,7 @@
 Cohort 27 - Andela
 This is a project to book meals in a hotel, with ability of adding multiple user and real time change of events on the web application.
 It combines html, css and Js so far
+Creating a web application, to fit meals ordering market specificaion
+following are my handles:
+https://www.pivotaltracker.com/n/projects/2167130
+https://github.com/Davincier/Book-A-Meal
```

Comment:
add screenshots of your app and a description of what your app does, plus explain to a new user who wants  to use your app how to navigate around your app